1.1/learn/how-to-keep-ballerina-up-to-date/index.html

```diff
@@ -255,7 +255,7 @@ <h2 id="terminology">Terminology</h2>
 
```

Comment:
Shouldn't we update L249?

"Update the Ballerina Tool"

Reply:
Done.